www_src/pages/login/sign-in.jsx

```diff
@@ -79,6 +79,9 @@ var SignIn = React.createClass({
     api.authenticate({json}, (err, data) => {
       this.props.setParentState({loading: false});
       if (err) {
+        if (window.Android) {
```

Comment:
In this case it doesn't matter since login only happens in one place, but if you'd like to hook into the api methods directly, they're here: https://github.com/mozilla/webmaker-android/blob/develop/www_src/lib/api.js#L133